recipes/wasmtime/all/test_package/CMakeLists.txt

```diff
@@ -1,13 +1,11 @@
 cmake_minimum_required(VERSION 3.1)
-project(PackageTest C)
-
-set(CMAKE_C_STANDARD 11)
```

Comment:
Why not to use `set(CMAKE_C_STANDARD 11)` at top of file ?

Reply:
It's an idiom we adopted at cci.
"Modern CMake" is all about using targets and avoiding global variables.

It probably does not matter for such a small example, but most of us believe it's better to use best practices as much as possible.